tests/publiccloud/instance_overview.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
```

Comment:
@cfconrad , I have a feeling that we already have all or most part of this info somewhere ? img_proof tests ? or we gathering this info somewhere ? terraform output ?

Reply:
This whole test is actually not a test but bunch of commands I used to proof that the SSH console works. I would personally keep it just to see something basic at the beginning but we can remove it as well...

Reply:
Some of images available in Public Cloud expecting something like this - they called BYOS - there you getting unregistered image which suppose to register . But others will give you image which already registered via Public Cloud Provider so you don't need to register it and can just use it. Not sure how this type of images will react on attempt to register them via SCC ...

Reply:
@asmorodskyi you are right that img_proof collects some of it and so on. But I actually don't know if QAM team will run img_proof at all. From what I see currently, they are using a different approach to run openQA modules on the instance. I would keep it as well.